hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/exceptions/SCMException.java

```diff
@@ -124,6 +124,7 @@ public ResultCodes getResult() {
     FAILED_TO_ALLOCATE_ENOUGH_BLOCKS,
     INTERNAL_ERROR,
     FAILED_TO_INIT_PIPELINE_CHOOSE_POLICY,
-    FAILED_TO_INIT_LEADER_CHOOSE_POLICY
+    FAILED_TO_INIT_LEADER_CHOOSE_POLICY,
+    FAILED_TO_FINALIZE_UPGRADE
```

Comment:
I don't see any usages for this.

Reply:
yes, left over code when I added UpgradeException. I will remove this.